identity/backend/src/main/resources/schema.sql

```diff
@@ -48,7 +48,5 @@ CREATE TABLE IF NOT EXISTS group_members (
 );
 
 CREATE TABLE IF NOT EXISTS profiles(
```

Comment:
How will it behave if multiple nodes try to create the schema simultaneously? Is there any locking applied?

Reply:
We haven't specifically configured locking (the creation of the schema is left to the inner working of Spring), this is something we should consider but I'd rather address it as a follow up to unblock others with this PR